app/decorators/article_decorator.rb

```diff
@@ -86,4 +86,14 @@ def video_metadata
   def long_markdown?
     body_markdown.present? && body_markdown.size > LONG_MARKDOWN_THRESHOLD
   end
+
+  def co_authors
+    User.where(id: co_author_ids)
```

Comment:
small optimization, as it seems you're only using `path` and `name`, maybe you can select only the needed fields: `username` and `name`. 

```ruby
 User.select(:name, :username).where(id: co_author_ids)
```

this so we don't load the entire row for only what we need

Reply:
Great suggestion!